docs/commands/dev.md

```diff
@@ -67,6 +68,36 @@ netlify dev:exec
 $ netlify dev:exec npm run bootstrap
 ```
 
+---
+## `dev:trace`
+
+Trace command
+Simulates Netlify's Edge routing logic to match specific requests. 
+This command is designed to mimic cURL's command line, so the flags are more familiar.
+
+
+**Usage**
+
+```bash
+netlify dev:trace
+```
+
+**Flags**
+
+- `debug` (*boolean*) - Print debugging information
+- `httpProxy` (*string*) - Proxy server address to route requests through.
```

Comment:
The flags are not correct, I guess they are copied from somewhere else. These are the flags:

```
 - `debug` (*boolean*) - Print debugging information 
- `watch` (*string*) - Path to the publish directory.
- `header` (*string*) - Request header, this flag can be used multiple times. Example: "Host: netlify.test"
- `cookie` (*string*) - Request cookie, this flag can be used multiple times. Example: "nf_jwt=token"
```

Reply:
Good catch, these are default flags shared by all commands